pkg/apis/enterprise/v3/common_types.go

```diff
@@ -276,6 +276,18 @@ type AppFrameworkSpec struct {
 	//    3. If anything more than the max value is specified then we set it to 1 day.
 	AppsRepoPollInterval int64 `json:"appsRepoPollIntervalSeconds,omitempty"`
 
+	// Period after which the scheduler tries to yield from the current reconcile entry
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default:=90
+	// +kubebuilder:validation:Minimum:=30
```

Comment:
is this auto-generated or done manually?

Reply:
It is added manually.